pkg/labeler.go

```diff
@@ -44,7 +44,8 @@ func NewTitleCondition() Condition {
 				return false, fmt.Errorf("title is not set in config")
 			}
 			log.Printf("Matching `%s` against: `%s`", matcher.Title, pr.GetTitle())
-			isMatched, _ := regexp.Match(matcher.Title, []byte(pr.GetTitle()))
+			re := regexp2.MustCompile(matcher.Title, 0)
```

Comment:
Use [`Compile`](https://godoc.org/github.com/dlclark/regexp2#Compile) instead of `MustCompile`